### PR TITLE
feat(connector): connector error handler, throw errmsg on general errors

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Update connector error handler, throw error description on General errors.

## Testing

UTs and tested locally.

### CODE REVIEW

File: packages/connector-alipay/src/index.ts [hunk 1]
```diff
@@ -137,36 +137,46 @@ export default class AlipayConnector implements SocialConnector {
...
-    assert(code !== '20001', new ConnectorError(ConnectorErrorCodes.SocialAccessTokenInvalid, msg));
-    assert(
-      sub_code !== 'isv.code-invalid',
-      new ConnectorError(ConnectorErrorCodes.SocialAuthCodeInvalid, msg)
-    );
-    assert(!sub_code, new ConnectorError(ConnectorErrorCodes.General, msg));
+  private readonly errorHandler: ErrorHandler = (response) => {
+    const { code, msg, sub_code, sub_msg } = response;
+
+    if (code === '20001') {
```
[the remark this answers is not in the document]
Comment (gao-sun) [reply packages/connector-alipay/src/index.ts:164]: it'll be better to have a const for this instead of a magic number/string

**Review by gao-sun — COMMENTED** [review]
let's paste the screenshot next time if the code itself cannot show the result clearly

**Comment by gao-sun** [comment]
also, remember to use squash merge :-)

File: packages/connector-alipay-native/src/index.test.ts [hunk 2]
```diff
@@ -216,7 +216,12 @@ describe('getUserInfo', () => {
       });
 
     await expect(alipayNativeMethods.getUserInfo({ auth_code: 'wrong_code' })).rejects.toMatchError(
-      new ConnectorError(ConnectorErrorCodes.General, 'Invalid parameter')
+      new ConnectorError(ConnectorErrorCodes.General, {
+        errorDescription: 'Invalid parameter',
+        code: '40002',
```
Comment (wangsijie) [inline packages/connector-alipay-native/src/index.test.ts:221]: where is this code come from?

File: packages/connector-alipay/src/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (darcyYe) [reply packages/connector-alipay/src/index.ts]: Seems that you are right, let's check this later and see if one Alipay mobile app can also work on desktop web.

**Review by gao-sun — APPROVED** [review]
looks good to me. but @wangsijie would have more context. better to have another stamp

**Pull request merged**